src/index.js

```diff
@@ -19,16 +21,16 @@ const Index = () => {
     level: "",
     tags: [],
     creator: "",
-    language: ""
+    language: "",
   });
 
   const value = { searchTerm, setSearchTerm, filterQuery, setFilterQuery };
   return (
-    <React.StrictMode>
+    <Provider store={store}>
```

Comment:
@Angryman18 We still need the strict mode right?

Reply:
StrictMode actually calling api's twice so i removed it. we can add again